easybuild/easyconfigs/m/multichoose/multichoose-1.0.3-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'multichoose'
+version = '1.0.3'
+
+homepage = 'https://github.com/ekg/multichoose'
+description = """generate multiset combinations (n multichoose k)."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['74f0a223c670f5aa81b14191c53ad8d84281838a47471c10253ae391f736c877']
+
+builddependencies = [('binutils', '2.36.1')]
+
+local_bins = [name, 'multipermute']
+files_to_copy = [(local_bins, 'bin'), (['*.h'], 'include/multichoose')]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_bins],
+    'dirs': [],
+}
+
```

Comment:
@verdurin Though `multichoose --help` doesn't really seem to work (it returns exit code 1), would it make sense to use the example from their README.md as sanity check command?
```
multichoose 2 a t g c
a a 
a t 
a g 
a c 
t t
t g 
t c 
g g 
g c 
c c
```
Or even shorter, `multichoose 1 a`?

Reply:
@bedroge agreed - now added.